### PR TITLE
fixed more than 50 names in Spanish

### PR DESCRIPTION
**Pull request by timeline24**
checked with nwt reference document

### CODE REVIEW

**Comment by kreier** [comment]
Well done, thank you!

File: db/dictionary_es.csv [hunk 1]
```diff
@@ -1,398 +1,398 @@
...
+Methuselah,Matusalén,Methuselah,Luke 3:37
+Lamech,Lamec,Lamech,
+Noah,Noé,Noah,
+Shem,Sem,Shem,
+Arpachshad,Ar·pajshad,Arpachshad,Luke 3:36
+Shelah,Sela,Shelah,
+Eber,Éber,Eber,
+Peleg,Péleg,Peleg,
+Reu,Reu,Reu,
+Serug,Serug,Serug,Luke 3:35
```
Comment (kreier) [inline db/dictionary_es.csv:40]: Thanks for the name updates and inclusion of reference locations

File: db/dictionary_es.csv [hunk 2]
```diff
@@ -1,398 +1,398 @@
...
-Zedekiah,Sedekia,Zedekiah,
-Jeroboam,Botella grande,Jeroboam,
-Nadab,Nadab,Nadab,
-Baasha,Baasa,Baasha,
-Elah,Ela,Elah,
-Zimri,Zimrí,Zimri,
-OmriTibni,Omri y Tibni juntos,Omri and Tibni together,
-Omri,Omrí (solo),Omri (alone),
-Ahab,Acab,Ahab,
-Ahaziah,Ocozías,Ahaziah,
```
Comment (kreier) [inline db/dictionary_es.csv:92]: Ahaziah seems to be twice in the dictionary

Comment (timeline24) [reply db/dictionary_es.csv:92]: Along with Korah, Nahor and Zephaniah

**Pull request merged**